exercises/yacht/cases_test.go

```diff
@@ -0,0 +1,169 @@
+package yacht
+
+// Source: exercism/problem-specifications
+// Commit: 4bb9c29 Update canonical-data.json
```

Comment:
Running the generator makes the // Commit: ... line come out different.  So might as well push a commit with the latest copy to get it in sync.

Reply:
That's because my changes to problem-specifications were squashed before merging. My branch is now "18 commits ahead and one commit behind". 

What is the git invocation to get my fork realigned with the upstream? (I fixed manually by copying the commit message off of github, that is surely not the Right Way)

Reply:
Locate the older commit in your problem-specifications tree which is  also in the master branch on github.  Then just reset with the hard option using that commit id.

Likely something like:
```
git checkout master
git reset --hard 6a42552060affe0f7745d89d9f719df8f830653b
```
Then do a git fetch followed by git pull.